servicex/transformer_manager.py

```diff
@@ -227,6 +227,11 @@ def shutdown_transformer_job(request_id, namespace):
             namespace=namespace
         )
 
+        api_v1 = client.CoreV1Api()
```

Comment:
It would be best not to re-assign this variable, but name this something different like `core_api`

Reply:
fixed now